transifex/native/django/templatetags/transifex.py

```diff
@@ -209,7 +209,10 @@ def render(self, context):
         )
         if self.tag_name == "t":
             source_icu_template = escape_html(source_icu_template)
-            translation_icu_template = escape_html(translation_icu_template)
+            if translation_icu_template is not None:
```

Comment:
I think we need to reorder lines 211 - 212 --> should be safer and makes more sense, right? 
Actually we can have one line condition if at line 210 by combining both conditions in an `and` clause.

Reply:
Uhm, if `t` is used and there is no translation available, we should fall back to the **escaped** source string. Or at least we should make the escaped source string available to the missing policy to figure out what to do. no?